components/form/form-errory-summary.js

```diff
@@ -74,6 +74,7 @@ class FormErrorSummary extends LocalizeCoreElement(RtlMixin(LitElement)) {
 					<d2l-button-icon
 						aria-expanded=${this._expanded ? 'true' : 'false'}
 						icon=${this._expanded ? 'tier1:arrow-collapse-small' : 'tier1:arrow-expand-small' }
+						text="Error Summary"
```

Comment:
We'd want to add a lang term for this. e.g., [this PR](https://github.com/BrightspaceUI/core/pull/4780/files) adds a few new lang terms.